n4js-libs/packages/n4js-runtime/src/n4js/ReflectionBuilder.n4js

```diff
@@ -0,0 +1,437 @@
+/*
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+import {N4Type, N4Classifier, N4Class, N4Interface, N4EnumType, N4Annotation, N4DataField, N4Accessor, N4Method, N4Member} from 'N4BuiltInTypes';
+
+
+const SYMBOL_IDENTIFIER_PREFIX = "#";
+const INTERFACE_FIELD_INITS = "$fieldInits";
+const INTERFACE_DEFAULT_MEMBERS = "$defaultMembers";
+const INTERFACE_EXTENDS = "$extends";
+const CLASS_IMPLEMENTS = "$implements";
+const PROPERTY_N4TYPE = "n4type";
+
+
+export function $interfaceFieldInits(obj: any+) : Object {
```

Comment:
please remove the leading `$` from all functions that are only used inside the runtime and are not made available to the generated output code created by the transpiler.

I think it applies to:
```
$interfaceFieldInits
$interfaceDefaultMembers
$interfaceExtends
$classImplements
```
(but please double check if there are more, also in file `Runtime.n4js`)